nntrainer/utils/node_exporter.h

```diff
@@ -15,6 +15,7 @@
 #include <vector>
 
 #include <nntrainer_error.h>
+#include <parse_util.h>
```

Comment:
it seems like parse_util will now also be needed to export in devel header as node_exporter is needed by layer_internal.h

Reply:
Actually, we are only using `getKeyValue` and most of them can be deprecated, at the end of the day, we can make a decision to put `getKeyValue` as static here instead of exposing `parse_util`

Reply:
alright.